compute_endpoint/globus_compute_endpoint/engines/high_throughput/engine.py

```diff
@@ -243,7 +243,7 @@ def __init__(
         # Tuning info
         prefetch_capacity=10,
         provider=LocalProvider(),
-        address="127.0.0.1",
+        address="localhost",
```

Comment:
This is a default, not a requirement.  I suggest we leave it be.  In particular, the below code has always required it to be an IP address, whether IPv4 or IPv6.  (Enforced by the `ipaddress.ip_address(address)` call.)

Reply:
I wanted to remove any ipv4 specific defaults, to ensure there's no layers below that (only) works with ipv4

Reply:
reverted back to 127.0.0.1, as it should work in both ipv4 and v6 environments.